src/erros/validationError.js

```diff
@@ -0,0 +1,13 @@
+import invalidRequestError from "./badRequestError.js";
+
+class validationError extends invalidRequestError {
+  constructor(err) {
+    const errorMessages = Object.values(err.errors)
+      .map((err) => err.message)
+      .join("; ");
+
+    super(`Os seguintes erros foram encontrados: ${errorMessages}`);
+  }
```

Comment:
Aqui é mais uma opinião minha, e algo que podemos discutir e chegar em um consenso. Mas considerando manter a mesma estrutura dos demais erros, acredito que seja papel de quem chama o validationError de montar a mensagem, e o erro deve receber apenas a mensagem:

```suggestion
  constructor(message: "Houve um erro na validação dos dados enviados") {
     super(message);
  }
```

O que você acha?